website/docs/Examples/EMGProjects/EMG_Joystick.md

```diff
@@ -0,0 +1,67 @@
+---
+id: EMG_Joystick
+title: EMG Joystick
+---
+This tutorial will show you how to control a joystick using EMG data with the OpenBCI GUI. You can then use this muscle activated joystick for any purpose of your choosing! We have developed an tuneable adaptive EMG classifier for accurate control of the joystick. We used this to control a drone as seen in the 2023 OpenBCI TED Talk.
+
+Check out an example video of this joystick being put into action!
+
+<!-- [![EMG Joystick Tutorial](../../assets/TutorialImages/)]() -->
+
+## Materials Required
+
+1.  OpenBCI [Cyton Board](https://shop.openbci.com/collections/frontpage/products/cyton-biosensing-board-8-channel?variant=38958638542)
+2.  [Skintact sticky electrodes](https://shop.openbci.com/collections/frontpage/products/skintact-f301-pediatric-foam-solid-gel-electrodes-30-pack?variant=29467659395) or [IDUN Dryode](https://shop.openbci.com/collections/frontpage/products/idun-dryode-kit)
+3.  [EMG/ECG Snap Electrode Cables](https://shop.openbci.com/collections/frontpage/products/emg-ecg-snap-electrode-cables?variant=32372786958)
+4.  Computer with downloaded [OpenBCI GUI](Software/OpenBCISoftware/01-OpenBCI_GUI.md)
+
+## Step 1: Hardware Assembly
+
+Follow the [EMG Getting Started tutorial](GettingStarted/Biosensing-Setups/02-EMG-Setup.md) to connect the electrodes to your body and the Cyton board, and read data from it using the OpenBCI GUI. For this tutorial, you will need to connect up to four channels. The number of channels depends on how many degrees of freedom needed. Four channels will enable control of both positive and negative X/Y directions of the joystick. Only one ground is needed.
```

Comment:
to connect the electrodes to your body and the Cyton board. We will read data from the Cyton using the OpenBCI GUI.